src/test/java/com/awesomeorg/airlineservice/integration_tests/ReservationControllerIntegrationTest.java

```diff
@@ -0,0 +1,119 @@
+package com.awesomeorg.airlineservice.integration_tests;
+
+import com.awesomeorg.airlineservice.AbstractIntegrationTest;
+import com.awesomeorg.airlineservice.entity.Passenger;
+import com.awesomeorg.airlineservice.entity.Reservation;
+import com.awesomeorg.airlineservice.protocol.CreatePassengerRequest;
+import com.awesomeorg.airlineservice.protocol.CreateReservationRequest;
+import com.awesomeorg.airlineservice.repository.PassengerRepository;
+import com.awesomeorg.airlineservice.repository.ReservationRepository;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.SneakyThrows;
+import org.junit.Test;
```

Comment:
Pay attention at the library version. This is JUnit 4, we are using 5. Import correct one. your tests are not working